src/app/how-council-works/page.tsx

```diff
@@ -256,6 +326,42 @@ export default function HowCouncilWorks() {
             </span>
           </SectionText>
         </Section>
+        <Section>
+          <SectionHeading>Stay Informed, Take Action</SectionHeading>
+          <SectionText>
+            Now that you know how City Council works, why not take the next
+            step? Stay informed and make your voice heard! Check out our{' '}
+            <Link href="/actions" className="classic-link">
+              Actions Page
+            </Link>{' '}
+            to see upcoming Council items and take action in just one click.
+            <br />
+            <br />
+            Or visit our{' '}
+            <Link href="/councillors" className="classic-link">
+              Councillors Page
+            </Link>{' '}
+            to see how your Councillor has voted and whether they align with
+            your priorities. Your engagement can help shape the decisions that
+            impact Toronto—get involved today!
+          </SectionText>
+        </Section>
+        <Section>
+          <SectionText>
+            <span className="italic">
+              Illustrations by Luisa Castillo Henao. View more of her work on{' '}
+              <Link
```

Comment:
For this we should use `ExternalLink` that way it goes to a new tab and adds the rel option to prevent reverse tabnabbing